codebundles/terraform-cloud-workspace-lock-check/runbook.robot

```diff
@@ -0,0 +1,80 @@
+*** Settings ***
+Documentation       Check whether the Terraform Cloud Workspace is in a locked state.
+Metadata            Author    Nitin Madhok (nmadhok@icloud.com)
```

Comment:
At the current time, the author name should be a GitHub handle, as this is how the author is rendered in RunWhen Local.

Reply:
@stewartshea Updated as requested

Reply:
Thanks!